src/contacts.cs

```diff
@@ -515,6 +515,7 @@ interface CNContactVCardSerialization {
 		CNContact [] GetContactsFromData (NSData data, out NSError error);
 	}
 
+#if !XAMCORE_4_0
```

Comment:
Please mark each member with an `[Obsolete ("Use XXX instead")]`. That's the best way for developer to be ready if we ever have to break API in the future.